### ISSUE TITLE
is kafka-python deprecated?

### ISSUE THREAD

**Issue by RJ3** [dpkp/kafka-python#2290]
No release in 1.5 yrs. Just wondering if eventually changes on Kafka broker will become incompatible with what we have. Is the project dead?

**Comment by sparklecast1** [comment]
Damn, It would be painful.

**Comment by dpkp** [comment]
Im not doing new feature work currently. I am not aware of serious
compatibility issues, apart from new broker features that are not supported.

Bigger picture, there were 2 of us actively maintaining this project for
several years. Both have since moved to companies that do not use this
library. If there are others who are actively using and would like to join
the maintainer team, please reach out!

**Comment by fenchu** [comment]
Others are not able to get wheels for 3.10 out (https://github.com/confluentinc/confluent-kafka-python), so I really appreciate You keep up the work. I'll try and contribute if I can. I have a pretty complex ssl rig that triggers lots some bugs.

**Comment by akira-kurogane** [comment]
Please, be honest and declare if this project is deprecated.

I was building another project which uses the current committed code in the main branch. It's requirements install step pulled the latest released version 2.0.2, but then I was debugging 'Unrecognized configs: .... KafkaClient' messages for quite a while before I had the idea to check if the erroring code line, committed ~2 years ago, was in fact not part of the latest release.

Is another package recommend in place of this one?

**Comment by grofte** [comment]
Aiven is recommending the use of this package: https://docs.aiven.io/docs/products/kafka/howto/connect-with-python.html

They should really put some resources into maintaining it.

**Comment by hrichards** [comment]
@dpkp Let me know if you're still in need of maintainers (email in profile).  My company uses Kafka and Kafka-Python extensively and has need of some of the unreleased and in-PR functionality for this project, and so could surely come up with a maintainer or two.

Alternately, have you considered moving this project to Jazzband?  https://github.com/jazzband

**Comment by jeffwidman** [comment]
@hrichards your best bet is probably to directly email @dpkp (you can pull his email off one of his old commits).

**Comment by wbarnha** [comment]
I reached out to @dpkp, if I don't hear back, I'm probably going to start working on building workflows in https://github.com/wbarnha/kafka-python-ng and automating creating pull requests from the existing ones here.

**Comment by wbarnha** [comment]
Got push access, hopefully I'll get to revive this project shortly!

**Comment by wbarnha** [comment]
We're back!

**Comment by wbarnha** [comment]
I'm still waiting on getting PyPi workflows ready, in case anyone revisits this.

**Comment by wbarnha** [comment]
Refer to this for now: https://github.com/wbarnha/kafka-python-ng
```bash
pip3 install kafka-python-ng
```
